pkg/gds/certman/certs.go

```diff
@@ -779,19 +782,30 @@ vaspsLoop:
 				// we failed to send the email.
 				continue vaspsLoop
 			}
+			updateRequired = true
 
 		// Thirty days before expiration, send the reissuance reminder to the VASP and the TRISA admin.
 		case daysBeforeExpiration <= 30:
+			// If the reminder fails do not stop processing and attempt to send reminder to admins
 			if err = c.email.SendContactReissuanceReminder(vasp, 30, reissuanceDate); err != nil {
 				log.Error().Err(err).Str("vasp_id", vasp.Id).Msg("error sending thirty day reissuance reminder")
+			} else {
+				updateRequired = true
```

Comment:
I'm not sure this is correct, we have some internal logic in this method which means we may or may not send the email, and therefore the log is not necessarily always updated.

Reply:
Ah ok - glad you reviewed this PR! I'll see if I can get in the proto.Equal check.